src/styles/landing-page.ts

```diff
@@ -5,6 +5,18 @@ const grid: SxStyleProp = {
   background: '#ffffff',
 }
 
+const divider: SxStyleProp = {
```

Comment:
Maybe here at the hr selector you could include the `'border: none'` and `'color: ##E7E9EE'` attributes, since the hr tag default styles may look like some 3-dimensional divider using a default color (darker than the expected) if they are not explicitly defined ([reference here](https://electrictoolbox.com/style-html-hr-tag-css/#:~:text=Default%20style%20for%20the%20hr,it%20appear%20somewhat%203%2Ddimensional.)).

![image](https://user-images.githubusercontent.com/39717968/185647859-fca87b60-7fd6-4693-b22a-2969f223bda5.png)